src/sinks/console/config.rs

```diff
@@ -67,6 +67,10 @@ impl SinkConfig for ConsoleSinkConfig {
     fn sink_type(&self) -> &'static str {
         "console"
     }
+
+    fn can_acknowledge(&self) -> bool {
```

Comment:
I wonder if this could be `true` too? Could the `println` return, but something still interrupt it before it writes to the screen?

Reply:
For the `console` sink, the acknowledgement is happening implicitly through dropping the metadata when the event is encoded, instead of after the event is written. So it doesn't really support acknowledgements at this point.

Reply:
🤔 don't all sinks encode before they send? Would this issue be more wide-spread? Or what makes this sink different?

cc/ @pablosichert too as this is something to consider during the sink encoding work. We shouldn't drop the events until after they've been successfully sent.

Reply:
Ah, I see. My other comments here could be wrong then since, presumably, there may be sinks that are capable of acknowledging, but do not take the finalizers to do it properly and drop the events early.

I think this'll require a tedious survey 😄

Reply:
Exactly. 😢